src/common/axes/AxisTicks.jsx

```diff
@@ -221,7 +221,11 @@ module.exports = React.createClass({
               textAnchor={textAnchor}
               {...optionalTextProps}
             >
-              {tickFormat(tick)}
+              {`${tickFormat(tick)}`.split('\n').map((tickLabel) => (
```

Comment:
`{tickFormat(tick)}` should be enough instead wrapping it with template string ?